project/ProjectPlugin.scala

```diff
@@ -124,6 +124,10 @@ object ProjectPlugin extends AutoPlugin {
       scalaVersion := V.scala212,
       crossScalaVersions := Seq(V.scala211, V.scala212),
       scalacOptions ~= (_ filterNot Set("-Xlint").contains),
+      scalacOptions ++= ((scalaBinaryVersion.value, getEnvVar("SCALAENV")) match {
```

Comment:
Is this still necessary?

Reply:
@bilki I've just launched a new build process in Travis without these, to verify if the stack size fix did the trick by itself... 🤔

Reply:
@bilki yup, that did the trick! Thanks for the suggestion!!